docs/index.md

```diff
@@ -0,0 +1,19 @@
+# Welcome to Wagtail Content Import's Documentation
+
+Wagtail Content Import is a module for Wagtail that provides functionality for importing page content from 
+third-party sources. Page content is imported into a StreamField, using a set of customisable mappings.
```

Comment:
"for Wagtail that provides functionality for importing page content from third-party sources" -> "for importing page content into Wagtail from third-party sources"

Reply:
Yes, that sounds better, thanks - I've updated in the latest version.